bin/s2nc.c

```diff
@@ -70,6 +70,10 @@ void usage()
     fprintf(stderr, "    Drop and re-make the connection using Session ticket. If session ticket is disabled, then re-make the connection using Session-ID \n");
     fprintf(stderr, "  -T,--no-session-ticket \n");
     fprintf(stderr, "    Disable session ticket for resumption.\n");
+    fprintf(stderr, "  -o, --ticket-out [file path]\n");
+    fprintf(stderr, "    Path to a file where the session ticket can be stored.\n");
+    fprintf(stderr, "  -I, --ticket-in [file path]\n");
+    fprintf(stderr, "    Path to session ticket file to resume connection.\n");
```

Comment:
IMO if I were to see `-o` and `-I` I would have no idea what they meant. Additionally one being capital while the other being lowercase is also not ideal.

Do we perhaps not add shot-opts for these commands? And I'll say it, do we perhaps remove short-opts from some of the other less used options?

Reply:
I'll remove the short-opts for ticket-in and ticket-out and see if everyone gets mad at me. TBH I don't know if we guarantee stability for these arguments, but I doubt we could remove the short-opts once they're in the code base.